src/session.ts

```diff
@@ -140,6 +142,7 @@ export class MediaSession {
         readonly streamType: MediaStreamType,
         public logPageEvent = false,
         public logMediaEvent = true,
+        public baseCustomAttributes = {}
```

Comment:
@alexs-mparticle Using ModelAttributes for the type seems fine. Do you know if there's a reason that the value for the dictionary doesn't accept booleans? The data collection end of the product supports booleans so I would think the SDK should accept that as well.

Existing Code:
`export interface ModelAttributes {
    [key: string]: string | number;
}`

Theoretical Code:
`export interface ModelAttributes {
    [key: string]: string | number | boolean;
}`

Reply:
@TheOnlyRantis It might be an oversight on our part when we first created this definition. Booleans should work, but most of our attributes are strings and numbers, so I would verify that adding a boolean value shows up as expected in your LiveStream